### PR TITLE
Add nodejs-6.2.0 to runtime

### PR DESCRIPTION
**Pull request by AndWac**
Updated to newest LTS nodejs version.
Added gcc-5.3 patch from alp.

### CODE REVIEW

File: components/runtime/node6/Makefile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2015 EveryCity Ltd. All rights reserved.
+# Copyright 2016 Alexander Pyhalov, Andreas Wacknitz
```
Comment (pyhalov) [inline components/runtime/node6/Makefile]: Please, remove my copyright

**Comment by AndWac** [comment]
Why? The patch is from you.

**Comment by pyhalov** [comment]
Too many authors for one file. If you really want it, put copyright on separate line.

**Comment by xen0l** [comment]
@andwac could you put the copyright on separate line as requested? I'll merge it afterwards.

**Pull request merged**